lib/cmr/dmmt_client.rb

```diff
@@ -0,0 +1,20 @@
+module Cmr
+  class DmmtClient < BaseClient
+    def dmmt_get_approved_proposals(params, token, host, port)
+      url = if Rails.env.development?
+              "http://localhost:3000/approved_proposals"
```

Comment:
Investigate alternatives

Reply:
Anything involving https gives certification verification errors.
mmt.localtest fails to connect on 80.
Mimicking the test set up gives a bad request error because a plain http request is sent to port 443.

Reply:
The next step, to me, is to investigate self-signed certs for dev.  Is that something we want to pursue?

Reply:
My opinion is that it is not necessary. I think we may want to add a comment that you may need to change the port depending on if you run rails s on a different port.